charts/thoras/values.yaml

```diff
@@ -60,6 +60,19 @@ thorasApiServer:
   port: 443
   logLevel: "debug"
 
+thorasApiServerV2:
+  enabled: false
+  containerPort: 8443
+  podAnnotations: {}
+  limits:
+    cpu: 1000m
+    memory: 1000Mi
+  requests:
+    cpu: 100m
+    memory: 100Mi
+  port: 443
+  logLevel: "debug"
```

Comment:
[pebble] Should we default to info instead of here so we don't have to remember to switch it back when we go to prod? The onus of setting debug can rest on the caller for now (i can help set this up in our values file), holla

Reply:
Just a heads up, this limit exists for every deployment. Not just the new api service.